### ISSUE TITLE
ASPNETZERO: Impersonation is not working in AZURE

### ISSUE THREAD

**Issue by mohammad-shadmehr** [aspnetzero/documents#138]
I have an ASPNETZERO application (v6.4.0.0) .NetCore + Angular. I have successfully deployed it to AZURE and all its features seems working fine but the Impersonation! The feature works fine while I run the app locally. I am also pretty sure that it use to work in azure before but all of the sudden has stopped working since this morning.

When I click the "Login as this user" option it redirects to "https://mywebsite/app/notifications?impersonationToken=dcb2e4f9-0813-4ce2-b6ca-2e66f76da0b4&tenantId=1" for a moment and then gets back to the admin account again. I suspect there is something wrong in the AccountAppService/Impersonate as the admin user does not get log out.

I wonder if there is any specific configuration is required in Azure in regards to Cache as the impersonation seems rely on it.

**Comment by maliming** [comment]
You can ask questions here.
https://support.aspnetzero.com/QA/Questions
https://github.com/aspnetzero/aspnet-zero-core/issues/new

**Comment by mohammad-shadmehr** [comment]
@maliming my 1 year subscription has been over and therefore cant ask my questions in those links!

**Comment by ismcagdas** [comment]
@mohammad-shadmehr no specific configuration actually. By default memory cache is used, so it shouldn't be a problem.

By the way, as far as I remember, impersonation url sholdn't be **app/notifications**. If you are sure that it was working before, you can check your source control history to see what has been changed recently.

For your further questions you can use https://stackoverflow.com/questions/tagged/aspnetboilerplate

**Comment by mohammad-shadmehr** [comment]
@ismcagdas Thanks for your message. I changed the path to be notification as I want the impersonator lands on the notification page. I noticed that the logic for the impersonation is located in the AppPreBootsrap in its run method and therefore it should work regardless the path I set for it right? It is working fine in my local and pretty sure used to work in azure before. Have already asked it in stackoverflow but no input!
